implementations/rust/ockam/ockam_node/src/router/record.rs

```diff
@@ -302,6 +303,7 @@ impl AddressRecord {
     }
 
     /// Check the integrity of this record
+    #[inline]
```

Comment:
How do you decide which methods to inline? I can see some small methods doing boolean checks, but is there any other "rule of thumb" for doing this? Should it be added to any other methods in the "hot path"?